docs/guides/hedgehog-engine/blueblur/files/xncp/anims.md

```diff
@@ -84,6 +84,10 @@ Looking at the remaining **CastAnimationData** entries, we can see that none of
 
 ![](./assets/anims/image6.png)
 
+## Unsolved Mysteries about Animations
+An animation's loop is defined in code, so it can't be changed, however **Field00** in **CastAnimationDataSubData** does seem to control how fast a loop occurs, where the value **1** seems to loop it with a delay, while **2** makes it loop smoother.

```

Comment:
It's probably better to use a different title here, like "Additional Notes".

Also:
> An animation's loop is defined in code, so it can't be changed, however (...)

should probably be changed to

> An animation can only be looped using code, so that can't be controlled via the XNCP. However, (...)

Reply:
Fixed